app/controllers/users_controller.rb

```diff
@@ -32,11 +37,45 @@ def update
     end
   end
 
-  def index; end
+  def index
+    @users = User.all
+
+    @users = @users.search_by_name(params[:name])
+                   .search_by_email(params[:email])
+                   .search_by_status(params[:activated])
```

Comment:
```suggestion
    @users = User.search_by_name(params[:name])
                   .search_by_email(params[:email])
                   .search_by_status(params[:activated])
```
thêm cái sort nữa vào em ơi